README.md

```diff
@@ -215,12 +222,12 @@ with dag:
     ingest_models = aql.render(dir_path + "/ingest_models", orders_table=raw_orders)
 ```
 
-### Passing on tables to subsequent tasks
-
 Once the `render` function completes, it returns a dictionary of all of the models based on the sql file name.
 If you have a sql file named `join_orders_and_customers.sql`, then the result would be stored in `model["join_orders_and_customers]`.
 
-Here is an example of passing a single model to a subsequent rendering (perhaps you want to separate your ingest and transforms).
+### Passing on tables to subsequent tasks
+
+The following DAG is an example of passing a single model to a subsequent rendering (perhaps you want to separate your ingest and transforms):
```

Comment:
The `**` example seems to be an exact copy of the previous example, pretty sure that's a typo.